README.md

```diff
@@ -54,6 +55,10 @@ See more in the [Anatomy of a Contract](https://docs.near.org/build/smart-contra
 
 - node >=14 <16.6.0 || >16.6.0
 - pnpm >=7
+- NEAR CLI
+```
+npm install -g near-cli
+```
```

Comment:
Where is it used? If we need CLI, we need to migrate to https://near.cli.rs, but I wonder if it should be just part of `package.json` dev-dependencies and create relevant `scripts` instead of installing it globally

Reply:
Part of the prerequisites in case the user wants to use the CLI.

Reply:
I will remove it